CircleSpawn/CircleSpawn/ViewControllers/CircleSpawnController.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+class CircleSpawnController: UIViewController, UIGestureRecognizerDelegate {
+
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .white
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        initView()
+    }
+    
+    func initView() {
+        let doubleTapGesture: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap(_:)))
+        doubleTapGesture.numberOfTapsRequired = 2
+        doubleTapGesture.delegate = self
+        view.addGestureRecognizer(doubleTapGesture)
+    }
+    
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer,
+             shouldRequireFailureOf otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+          return true
+    }
```

Comment:
Bardzo niefajna implementacja tej metody. Wszystkie gesty wymagają, żeby wszystkie inne failowały, co powoduje że prosty pan przestanie działać na przykład.

Zamiast tego powinniśmy zobaczyć, że `gestureRecognizer` to nasz double tap, a `otherGestureRecognizer` to nasz triple tap.